examples/default/main.c

```diff
@@ -47,6 +52,14 @@ int main(void)
     rtc_init();
 #endif
 
+#ifdef MODULE_GNRC
```

Comment:
Since `pktdump` is the top of the dependency tree here, I would take `MODULE_GNRC_PKTDUMP` here